### PR TITLE
fix: Update README to reflect current state of fuseml-installer

### PR DESCRIPTION
**Pull request by jsuchome**
This is a minimal version of what needs to be done so the README does not contain nonsense.

Obviously changes should be much bigger for some production level of documentation....

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,56 +85,45 @@ The basic FuseML workflow can be described as a GitOps type of workflow that sta
...
+* git
 
-### Push an application
+  Install the package provided by your OS. On SUSE Linux, call
+  ```bash
+  $ sudo zypper install git-core
+  ```
 
-Run the following command for any supported application directory (e.g. one of the applications inside the [examples directory](examples)).
+* helm
```
Comment (flaviodsr) [inline README.md:97]: what about kubectl?

File: README.md [hunk 2]
```diff
@@ -154,6 +143,12 @@ $ fuseml COMMAND --help
 
 ```
 
+### Usage
+
+After using `fuseml-installer` to install required components of FuseML, for actual work with FuseML one needs to use `fuseml-core` command line client.
+
+Go to [fuseml-core project page](https://github.com/fuseml/fuseml-core) to learn about CLI.
+
 ## Configuration
```
Comment (flaviodsr) [inline README.md:152]: Is that section still valid?

**Review by flaviodsr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jsuchome) [reply README.md:97]: good point

File: README.md (hunk 2 above)
Comment (jsuchome) [reply README.md:152]: I planned to look into that. I believe it still has its use, or that we can reuse it for something else. If I find out the opposite, I'll remove it as well.

**Review by flaviodsr — APPROVED** [review]

**Pull request merged**